recipe/meta.yaml

```diff
@@ -7,16 +7,15 @@ package:
   version: {{ version|replace("-", "_") }}
 
 source:
-  fn: GGally_{{ version }}.tar.gz
   url:
     - {{ cran_mirror }}/src/contrib/GGally_{{ version }}.tar.gz
     - {{ cran_mirror }}/src/contrib/Archive/GGally/GGally_{{ version }}.tar.gz
   sha256: 3ad00c8cd6738887b6cbe3450e8418ea1ba7dfd8985bc0e7f59b6f1fe0bf75be
 
 build:
-  noarch: generic
-  merge_build_host: true  # [win]
+  merge_build_host: True  # [win]
   number: 0
```

Comment:
Hm. Will that upload without bumping the build number?

Reply:
Gah, missed that. Will fix, thanks.